python/singa/autograd.py

```diff
@@ -631,16 +712,39 @@ def backward(self, dy):
 
 
 def add_bias(x, b, axis=0):
+    """
+    Add Bias to each row / column of the Tensor, depending on the axis arg.
+    Args:
+        x: matrix.
+        b (CTensor): bias to be added.
+        axis (int): 0 or 1, default is 0.
+    Return:
+        the result Tensor
+    """
     return AddBias(axis)(x, b)[0]
 
-
 class Reshape(Operation):
+    """
+    Reshape the input tensor similar to numpy.reshape. 
+    """
 
     def __init__(self, shape):
+        """
+        Args:
+            shape: Specified shape for output. At most one dimension of the new shape can be -1. In this case, 
```

Comment:
try to keep  the length of a line within 80.

Reply:
fixed, and I have checked the whole file again.